dataland-frontend/src/frameworks/sme/UploadConfig.ts

```diff
@@ -1,34 +1,34 @@
-import { type SmeData } from "@clients/backend";
 import { type Category } from "@/utils/GenericFrameworkTypes";
```

Comment:
Is this not generated? Where is the comment?

Reply:
Steht jetzt in allen autogeneriert.

Hab das devtools einfach einmal mit dem arg `all` laufen lassen.
Ich musste/wollte das sowieso mal machen nach den ganzen Änderungen. Also sollte jetzt passen 👍

Reply:
❤️